### DISCUSSION TITLE
Vue - Official extension: What has changed in v2.0.8?

### DISCUSSION THREAD

**Discussion by alwaqfi** [vuejs/language-tools#4185] [Q&A] [answered]
Hello,

VS Code shows that there is a new version of Vue - Official extension (v2.0.8) but I can't find anywhere the version and either what has changed. I look in the repository releases/tags/change logs but the latest version I found was v2.0.7.

So, I want to ask if this is a legit version and any idea what has changed?

Thanks,

**Comment by johnsoncodehk** [comment, marked as answer]
Since we can't publish the master branch bug fixes before VSCode 1.88 released, I forked 2.0.7 and released 2.0.8 just to [remove the insider edition status](https://github.com/vuejs/language-tools/commit/44ce1c89b923dda30840daa34b17d8b95c197a23). (Context: https://github.com/vuejs/language-tools/discussions/4127#discussioncomment-8910096)

**Reply by alwaqfi** [reply]
Thank you!
